src/steps/os/archlinux.rs

```diff
@@ -32,7 +31,10 @@ pub struct YayParu {
 impl ArchPackageManager for YayParu {
     fn upgrade(&self, ctx: &ExecutionContext) -> Result<()> {
         if ctx.config().show_arch_news() {
-            Command::new(&self.executable).arg("-Pw").status_checked()?;
+            ctx.run_type()
+                .execute(&self.executable)
+                .arg("-Pw")
+                .status_checked_with_codes(&[1, 0])?;
```

Comment:
May it show “success” when this command failed with return code 1?

Reply:
How do you mean that?

Reply:
This will consider the command (`paru -Pw` or `yay -Pw`) to have succeeded if it exits with return codes 1 or 0. It won't print "success" but it will continue to execute `topgrade`. (It won't fail.)

Reply:
I don't see a reason why we should print a success because it still errors out if something goes wrong.